CHANGES.rst

```diff
@@ -4,6 +4,8 @@ astropy-helpers Changelog
 3.2 (unreleased)
 ----------------
 
+- Avoid importing whole package when trying to get version information. [#442]
```

Comment:
Please mention the minor api breakage of `cython_version` and `compiler` not being available any more